modules/Bio/EnsEMBL/Compara/Utils/IDGenerator.pm

```diff
@@ -0,0 +1,125 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl help desk at
+<http://www.ensembl.org/Help/Contact>.
+
+=head1 METHODS
+
```

Comment:
I thought we didn't put the `CONTACT` field any more? And maybe adding a `DESCRIPTION` field would be helpful for newcomers? (although the module name is already quite descriptive)
```suggestion
```

Reply:
```suggestion
=head1 DESCRIPTION

Helper function to assign unique numeric identifiers even when the data are
not stored in the database. This is primarily intended for pipelines that
first store their data in flat files and load them later into the database.

=head1 METHODS
```